### DISCUSSION TITLE
Ciphey just lock

### DISCUSSION THREAD

**Discussion by karlreyka** [bee-san/Ciphey#673] [Q&A] [answered]
I just install Ciphey on linux Ubuntu, python 3.8, Docker.
When I try the example: 
docker run -it --rm remnux/ciphey -t "aGVsbG8gbXkgbmFtZSBpcyBiZWU="

It is running correctly, and I have an answer.

When i try another code:
docker run -it --rm remnux/ciphey -t "rzLCBYPXWDcRKybFEzXM"

It run, and lock without any answer.
see attached image:
![Sans titre](https://user-images.githubusercontent.com/87601660/126063426-efceb83f-8b3b-4230-8dd7-01e2c65a48a4.png)

Thank's for your answers.

**Comment by SkeletalDemise** [comment, marked as answer]
If Ciphey gets stuck on "Thinking..." it usually means that Ciphey can't crack your code. `rzLCBYPXWDcRKybFEzXM` is not just Base64 like the example. I don't know what it is. It's very likely that Ciphey doesn't support whatever encoding/encryption that your code is using.

**Reply by karlreyka** [reply]
OK. Thank's for your answer.
Do you have an idea of what could be the way to crack this code ?

**Reply by SkeletalDemise** [reply]
I would need more context about the code. Where is the code from? What would the decrypted message be? Would it be in English or in another language? It's hard to crack it without any knowledge or background on it.

**Reply by karlreyka** [reply]
Thank's for your proposal. I have found another solution with a freelancer.
Regards

**Reply by GlenH9** [reply]
hey..same here.. can pass on the freelancer details, if you can?

**Comment by slezisatoshiz** [comment]
Good guys!

**Comment by AiPals** [comment]
> I just install Ciphey on linux Ubuntu, python 3.8, Docker. When I try the example: docker run -it --rm remnux/ciphey -t "aGVsbG8gbXkgbmFtZSBpcyBiZWU="
> 
> It is running correctly, and I have an answer.
> 
> When i try another code: docker run -it --rm remnux/ciphey -t "rzLCBYPXWDcRKybFEzXM"
> 
> It run, and lock without any answer. see attached image: ![Sans titre](https://user-images.githubusercontent.com/87601660/126063426-efceb83f-8b3b-4230-8dd7-01e2c65a48a4.png)
> 
> Thank's for your answers.

**Comment by AiPals** [comment]
The code snippet aGVsbG8gbXkgbmFtZSBpcyBiZwU= appears to be a Base64-encoded string. Base64 is a common encoding scheme used to represent binary data in an ASCII string format. Decoding it will reveal the original text.

Decoding the Base64 String

Let’s decode aGVsbG8gbXkgbmFtZSBpcyBiZwU= to see what it represents.

	1.	Base64 Encoding:
	•	The string aGVsbG8gbXkgbmFtZSBpcyBiZwU= can be decoded from Base64.

Here is the Python code to decode it:
import base64

# Base64 encoded string
encoded_str = "aGVsbG8gbXkgbmFtZSBpcyBiZwU="

# Decode the Base64 string
decoded_bytes = base64.b64decode(encoded_str)
decoded_str = decoded_bytes.decode('utf-8')

print(decoded_str)
Decoding the String

When you run the above Python code, the decoded string is:
hello my name is bee
But I don’t think that’s the problem, to use Ciphey in a Docker container as mentioned:

docker run -it --rm remnux/ciphey -t "aGVsbG8gbXkgbmFtZSBpcyBiZwU="

This command will automatically attempt to decrypt the provided string using Ciphey’s capabilities. If the string is indeed encoded in Base64 or any other common encoding, Ciphey should be able to identify and decode it. 
I’m glad you were able to identify the problem and solve it!. I just wanted to share my findings.
If you need any assistance I’m here to help out.
